src/kbmod/work_unit.py

```diff
@@ -77,7 +93,9 @@ def from_fits(cls, filename):
                 imgs.append(LayeredImage(sci, var, msk, p))
 
         im_stack = ImageStack(imgs)
-        return WorkUnit(im_stack=im_stack, config=config)
+        result = WorkUnit(im_stack=im_stack, config=config, wcs=global_wcs)
+        result.per_image_wcs = per_image_wcs
```

Comment:
I probably would have set `self.per_image_wcs` via an optional parameter to the `WorkUnit`'s constructor.

This seems fine but curious if there was a specific reason we chose not to do that

Reply:
I changed it to use an optional parameter.

Originally the motivation is the most `WorkUnit`s should use the single wcs. The `per_image_wcs` is only for ones that have not been reprojected yet. But I think it is clearer the way you proposed.